### ISSUE TITLE
InputStream.available() returns 0

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dorkasdev/bluecove#11]
```
InputStream.available() not implemented


```

Original issue reported on code.google.com by `skarzhev...@gmail.com` on 24 Apr 2007 at 3:08

**Comment by GoogleCodeExporter** [comment]
```
Implemented using ioctlsocket in version 1.2.3
```

Original comment by `skarzhev...@gmail.com` on 24 Apr 2007 at 3:09

**Comment by GoogleCodeExporter** [comment]
Original comment by `skarzhev...@gmail.com` on 11 May 2007 at 6:07
- Changed state: **Fixed**
